js/util/ajax.js

```diff
@@ -59,7 +61,11 @@ exports.getImage = function(url, callback) {
             (window.URL || window.webkitURL).revokeObjectURL(img.src);
         };
         var blob = new window.Blob([new Uint8Array(imgData)], { type: 'image/png' });
-        img.src = (window.URL || window.webkitURL).createObjectURL(blob);
+        if (imgData.byteLength) {
```

Comment:
It is unfortunate that we don't have any unit tests for this module yet 😬. I suggest that we create a `test/js/util/ajax.test.js` file and add at least a test for this case. Sinon's [`window. useFakeXMLHttpRequest`](http://sinonjs.org/docs/#server) should make it relatively easy to simulate this case.